src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_traces_overview/sub_components/trace_summary.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { EuiHorizontalRule } from '@elastic/eui';
+import React from 'react';
+import { TRANSACTION_NAME_FIELD } from '@kbn/discover-utils';
+import { useTransactionContext } from '../../../hooks/use_transaction';
+import { FieldWithActions } from './field_with_actions/field_with_actions';
+
+export interface TraceSummaryProps {
+  fieldId: string;
+  fieldConfiguration: any;
+}
+
+export function TraceSummary({ fieldConfiguration, fieldId }: TraceSummaryProps) {
+  const { transaction } = useTransactionContext();
+
+  if (fieldId === TRANSACTION_NAME_FIELD) {
+    if (transaction) {
+      fieldConfiguration.value = transaction.name;
+    }
+  }
+
+  if (!fieldConfiguration.value) {
+    return null;
+  }
+
+  return (
+    <>
+      <FieldWithActions
+        data-test-subj={`unifiedDocViewTracesOverviewAttribute-${fieldId}`}
+        label={fieldConfiguration.title}
+        field={fieldId}
+        value={fieldConfiguration.value}
+        formattedValue={fieldConfiguration.value}
+        fieldMetadata={fieldConfiguration.fieldMetadata}
+      >
+        {() => <div>{fieldConfiguration.content(fieldConfiguration.value)}</div>}
```

Comment:
if you adjust FieldWithActionsProps you can just pass the child component like this
```suggestion
        <div>{fieldConfiguration.content(fieldConfiguration.value)}</div>
```

Reply:
https://github.com/elastic/kibana/pull/210612/commits/87a9b3e6789f5f591d91a37214c8a7f7bb1c49c1

Reply:
Sorry, I had to make some tweaks to your proposal since `content` from `getFieldConfiguration` is defined as `(value: any) => React.ReactNode`, so it needs to be invoked as a function to render correctly.

I also cleaned up `FieldWithActions` by removing leftover content from the original component.

https://github.com/elastic/kibana/pull/210612/commits/324fee6e9aa6083d129ad33ecb6f08584f60bc48